exercises/secret-handshake/instructions.md

```diff
@@ -2,8 +2,7 @@
 
 Your task is to convert a number between 1 and 31 to a sequence of actions in the secret handshake.
 
-The sequence of actions is chosen by looking at the rightmost five digits of the number once it's been converted to binary.
-Start at the right-most digit and move left.
+The sequence of actions is chosen by looking at the five digits of the number (which is between 1 and 31, equals 2 ^ 5 - 1) once it's been converted to binary. Start at the right-most digit and move left.
```

Comment:
We intentionally use one sentence per line. Please preserve that formatting.

Reply:
OK.